src/Core/Expressions/Identifier.cs

```diff
@@ -159,6 +159,8 @@ public MemoryIdentifier(string name, DataType dt, Storage stg) : base(name, dt,
         {
         }
 
+        public bool IsFpuStack => Name.StartsWith("ST"); //$HACK: gross hack but we have to start somewhere.
+
```

Comment:
I wish to avoid this X86 specific code in this General purpose class. See my other comment.